apache2/persist_dbm.c

```diff
@@ -14,17 +14,11 @@
 
 #include "persist_dbm.h"
 #include "apr_sdbm.h"
+#ifdef MEMORY_DATABASE_ENABLE
 #include "ag_mdb_external.h"
-#include "errno.h"
```

Comment:
why delete these code in the compiling change?

Reply:
I have moved it into ag_mdb_external.h